cost/aws/rightsize_ec2_instances/aws_rightsize_ec2_instances.pt

```diff
@@ -626,6 +639,14 @@ script "js_cloudwatch_queries", type: "javascript" do
         ]
       }
 
+      // If instance.tags contains object with key value == aws:autoscaling:groupName
```

Comment:
Could be refactored to be simpler while also avoiding errors in situations where the instance has no tags

```
if (typeof(instance['tags']) == 'object' && instance['tags']['aws:autoscaling:groupName']) { 
  dimensions.push({ "Name": "AutoScalingGroupName", "Value": asg_name['value'] })
}
```